### ISSUE TITLE
parsing links ending with brackets

### ISSUE THREAD

**Issue by razie** [tristanjuricek/knockoff#29]
eats up the last bracket...

```
object Test extends App {
  import com.tristanhunt.knockoff.DefaultDiscounter._
  println(toXHTML(knockoff("[wiki link](http://en.wikipedia.org/wiki/Bracket_(disambiguation))")).toString)
}
```

output - note the missing bracket in the href:

```
<p><a  href="http://en.wikipedia.org/wiki/Bracket_(disambiguation">wiki link</a>)</p>
```

**Comment by razie** [comment]
click "edit" to see the source of my comment above :) heh even this is markdown

**Comment by tristanjuricek** [comment]
Should be fixed in 0.8.1
